src/common/cast/base-remote-player.js

```diff
@@ -309,6 +310,10 @@ class BaseRemotePlayer extends FakeEventTarget implements IRemotePlayer {
     return true;
   }
 
+  isCastInitiator(): boolean {
+    return this._isCastInitiator;
```

Comment:
```suggestion
    return this._isCastInitiator || RemotePlayerManager._loadedPlayerCount === 1;
```
so you can remove the RemotePlayerManager.load

Reply:
it's a counter by type and we don't have a type in the base class.
also I don't like putting a specific logic in the base class in case the children want to override it, which is why i put the generic logic of returning _isCastInitiator.

Reply:
1. if the child has the type the base has it also (this refers to the child)
2. if a child wants to override it's still possible, just implement `isCastInitiator` (like you do for all)

Reply:
I don't like having this logic in the parent, I'll just implement it in the sender instead